core/jobs/types/firebase_domain_test.py

```
@@ -0,0 +1,195 @@
+        self,
+    ) -> None:
+        auth_details = auth_models.UserAuthDetailsModel(
+            id='uid_a',
+            firebase_auth_id=None,
+            deleted=False,
+            parent_user_id=None,
+        )
+        settings = user_models.UserSettingsModel(
+            id='uid_a', email='a@a.com', deleted=False
+        )
+        with self.assertRaisesRegex(ValueError, r'must have non-empty'):
```

Ruff (0.15.17)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)